### DISCUSSION TITLE
Accessing fields that contain a special character

### DISCUSSION THREAD

**Discussion by yxyx520** [vectordotdev/vector#11381] [Q&A] [answered]
json like blow, how to get the value of "b/c"?
```json
{
  "a": {
    "b/c": 2
  }
}
```
I try some transforms config blow but failed:
- `.a.b/c`
- `.a.'"b/c"'`
- `.a'."b/c"'`
- `.a["b/c"]`

most error like `unexpected end of query path`  

when I use jq to accessing fields I succeed
```shell
$ echo '{"a": {"b/c": 2}}' | jq .a.'"b/c"'
$ 2
$ echo '{"a": {"b/c": 2}}' | jq .a'."b/c"'
$ 2
```

In kubernetes, there are some lables with "/", such as "app_kubernetes_io/name" and "app_kubernetes_io/component", I do not find any method to get those values with "/" in key.

**Comment by jszwedko** [comment, marked as answer]
Answered over here: https://github.com/vectordotdev/vector/issues/11382#issuecomment-1040405722

**Reply by yxyx520** [reply]
thanks, it works
